pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -436,90 +439,829 @@ func Test_server_Exists(t *testing.T) {
 
 func Test_server_Search(t *testing.T) {
 	t.Parallel()
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
 	type args struct {
 		ctx context.Context
+		num int
```

Comment:
I suppose it'd better to use `insertNum` instead of it.
If it is used for only insert parameter.